src/pages/team/index.tsx

```diff
@@ -0,0 +1,73 @@
+/* eslint-disable @next/next/no-img-element */
+import React from "react";
+import TeamHero from "../../components/team/TeamHero";
+import TeamCard from "../../components/team/TeamCard";
+import { departmentsData } from "../../components/team/team-data";
+import FooterComp from "../../components/FooterComp";
+import { DepartmentsEnum } from "components/team/team-interfaces";
+import SubscriptionForm from "components/team/SubscriptionForm";
+
+const TeamPage = () => {
+	return (
+		<div className="team">
+			<TeamHero heroTitle="Our Team" />
+
+			<main className="container-fluid team-categories">
+				<div className="strokes">
+					<img src="/images/Team/Ellipse-right.png" alt="" id="ellipse-right" />
+					<img src="/images/Team/Ellipse-left.png" alt="" id="ellipse-left" />
+				</div>
+
+				<div className="desc">
+					<p id="rochester-p">Leadership</p>
```

Comment:
```jsx
<p id="rochester-p">Leadership</p>
```
Explain why you have to use ID here

Reply:
I noticed paragraphs that had rochester font all had a particular styling as seen on the figma element inspection.

So I styled them once using the id to aid re-usage of same styling on similar elements.